### PR TITLE
patch(disputer,liquidator): Handle changed LiquidationWithdrawn event params

### PR DESCRIPTION
**Pull request by nicholaspai**
Signed-off-by: Nick Pai <npai.nyc@gmail.com>

**Motivation**

Resolved disputes are withdrawn successfully but event log parsing is mishandled if EMP version is not a legacy (<=1.2.2) one.

Additionally, we can query `settlementPrice` from the event logs instead of making a call to the VotingContract.getPrice(), which removes the need for the bots to construct the Voting contract.

Tested log output to Slack successfully on Kovan.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [X]  Ran end-to-end test, running the code as in production
- [X]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/disputer/index.js [hunk 1]
```diff
@@ -89,7 +88,6 @@ async function run({
       );
 
     // Setup contract instances.
-    const voting = new web3.eth.Contract(getAbi("Voting", "1.2.2"), getAddress("Voting", networkId));
```
Comment (nicholaspai) [inline packages/disputer/index.js:92]: No more need to pass in `voting` to Disputer, which was previously used to make a `getPrice` call to figure out the settlement price. `settlementPrice` is part of event params!

File: packages/disputer/src/disputer.js [hunk 2]
```diff
@@ -230,7 +232,6 @@ class Disputer {
         at: "Disputer",
         message: "Position has been disputed!👮‍♂️",
         liquidation: disputeableLiquidation,
-        txnConfig,
```
Comment (nicholaspai) [inline packages/disputer/src/disputer.js:233]: No reason to print out `txnConfig`, remove log clutter

File: packages/disputer/src/disputer.js [hunk 3]
```diff
@@ -328,31 +322,30 @@ class Disputer {
...
-        .call({
-          from: this.financialContract.options.address
-        });
-
-      const logResult = {
+      let logResult = {
         tx: receipt.transactionHash,
         caller: receipt.events.LiquidationWithdrawn.returnValues.caller,
-        withdrawalAmount: receipt.events.LiquidationWithdrawn.returnValues.withdrawalAmount,
+        settlementPrice: receipt.events.LiquidationWithdrawn.returnValues.settlementPrice,
```
Comment (daywiss) [inline packages/disputer/src/disputer.js:328]: does this work with all previous contracts?

**Review by daywiss — APPROVED** [review]

File: packages/disputer/src/disputer.js (hunk 3 above)
Comment (nicholaspai) [reply packages/disputer/src/disputer.js:328]: Yep I double checked. You actually added this in a while ago (prior to version 1.1.0 even) but we never took advantage of it in the bots. This event change was made in July, a month before we set up our versioning!

https://github.com/UMAprotocol/protocol/pull/1753

File: packages/disputer/index.js (hunk 1 above)
Comment (mrice32) [reply packages/disputer/index.js:92]: Might be a dumb question, but won't we still need it if we're interacting with an older EMP?

Comment (mrice32) [reply packages/disputer/index.js:92]: Ah, saw your comment below about changing this a while back.

**Review by mrice32 — APPROVED** [review]
LGTM! Nice cleanup!

**Pull request merged**